src/main/java/frc/robot/subsystems/intake/ConstantsIntake.java

```diff
@@ -0,0 +1,10 @@
+package frc.robot.subsystems.intake;
+
+public class ConstantsIntake {
+    public static final double GEAR_RATIO = 1 / 26.07;
+    public static final double TICKS_PER_RADIAN = 42 * GEAR_RATIO / 2 * Math.PI;
```

Comment:
You might want to put parentheses around, I guess you wanted to divide by 2 pi, current you multiply by pi

Reply:
done.